honeybee_grasshopper_energy/src/HB Blend ProgramTypes.py

```diff
@@ -0,0 +1,54 @@
+# Honeybee: A Plugin for Environmental Analysis (GPL)
+# This file is part of Honeybee.
+#
+# Copyright (c) 2019, Ladybug Tools.
+# You should have received a copy of the GNU General Public License
+# along with Honeybee; If not, see <http://www.gnu.org/licenses/>.
+# 
+# @license GPL-3.0+ <http://spdx.org/licenses/GPL-3.0+>
+
+"""
+Create a ProgramType object by blending other ProgramTypes together using a weighted
```

Comment:
Thank you for the complement on the component and this is something that I went back and forth about.  It's true that it is most useful on the urban scale when you don't have time to break out individual zones but it also has a lot of usefulness in detailed modeling as well. It's kinda our answer right now to OpenStudio Spaces vs. Zones. If you want to group a bunch of ProgramTypes together into one Zone, you can create a blended ProgramType. Given this and the fact that I think it will be easier to find if it's grouped with the other components related to ProgramType, I'm leaning towards leaving it in honeybee-grasshopper-energy.